DxDispatch/src/dxdispatch/Executor.cpp

```diff
@@ -413,11 +428,26 @@ void Executor::operator()(const Model::WriteFileCommand& command)
 
     try
     {
-        auto resource = m_resources[command.resourceName];
-        auto fileData = m_device->Download(resource.Get());
         auto& resourceDesc = m_model.GetResource(command.resourceName);
         auto& bufferDesc = std::get<Model::BufferDesc>(resourceDesc.value);
 
+        std::vector<std::byte> fileData;
+        auto resource = m_resources[command.resourceName];
+        if (resource)
+        {
+            fileData = m_device->Download(resource.Get());
+        }
+        else
+        {
+            fileData = bufferDesc.initialValues;
```

Comment:
>fileData = bufferDesc.initialValues; [](http://example.com/codeflow?start=12&length=36)

will update the code to support deferred resource as well but currently onnx is providing only CPU tensor for this type, will update comment